WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -164,25 +164,33 @@ class DashboardPromptsCardCell: UICollectionViewCell, Reusable {
         return trainContainerView
     }
 
-    private var answerInfoLabel: UILabel {
-        let label = UILabel()
-        label.translatesAutoresizingMaskIntoConstraints = false
-        label.text = answerInfoText
-        label.font = WPStyleGuide.BloggingPrompts.answerInfoLabelFont
-        label.textColor = WPStyleGuide.BloggingPrompts.answerInfoLabelColor
-        label.textAlignment = (effectiveUserInterfaceLayoutDirection == .leftToRight ? .left : .right)
-        label.numberOfLines = 0
-        label.adjustsFontForContentSizeCategory = true
+    private var answerInfoButton: UIButton {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle(answerInfoText, for: .normal)
+        button.titleLabel?.font = WPStyleGuide.BloggingPrompts.answerInfoButtonFont
+        button.setTitleColor(WPStyleGuide.BloggingPrompts.buttonTitleColor, for: .normal)
```

Comment:
I suppose it should depend on `FeatureFlag.bloggingPromptsEnhancements.enabled` to make it grey

Reply:
Funny, I forgot to push the commit :] 
I've just did so, thanks for catching it!